src/calcligatureorneumeposfunctor.cpp

```diff
@@ -348,14 +348,14 @@ FunctorCode CalcLigatureOrNeumePosFunctor::VisitNeume(Neume *neume)
         }
 
         // If the nc overlaps with the previous, move it back from a line width
-        if (overlapWithPrevious) {
+        if (overlapWithPrevious && !m_doc->HasFacsimile()) {
             xRel -= lineWidth;
         }
 
         nc->SetDrawingXRel(xRel);
```

Comment:
Since `xRel` is not used with facsimile you should put everything in one block
```
        // xRel remains unset with facsimile
        if (!m_doc->HasFacsimile()) {
            // If the nc overlaps with the previous, move it back from a line width
            if (overlapWithPrevious) {
                xRel -= lineWidth;
            }
            
            nc->SetDrawingXRel(xRel);
            // The first glyph set the spacing - unless we are starting a ligature, in which case no spacing should be added
            // between the two nc
            if (!previousLig) {
                xRel += m_doc->GetGlyphWidth(nc->m_drawingGlyphs.at(0).m_fontNo, staffSize, false);
            }
        }
```